paywall/src/paywall-builder/build.js

```diff
@@ -17,19 +17,24 @@ const baseBannerHeight = window => {
   // can use it as the basis to scroll from
   return minHeightPct > 30 ? minHeightPct : 30
 }
+let scrollPolling = true
 // iOS allows the page to scroll even when the paywall is up. Our
```

Comment:
This actually reminds me that right now, the scrollPosition stuff is only relevant on iOS devices. I believe @julien51 was interested in making it the default behavior across the board. I guess we should decide whether that's still a goal and whether it justifies making an issue.

Reply:
I think that this should be default. Any device that allows scrolling should be fubared by the paywall

Reply:
Oh no, I agree with that. I think the discussion was around whether to always allow scrolling so that behavior is consistent across as many devices as possible instead of having iOS able to scroll (and the paywall grows to obscure) and having every other device stay frozen.

Reply:
ok. That's something to open an issue for, yes. This only fixes a potential problem where if the iframe is small, we don't want to post messages